data_scraping/tensorflow_data.py

```diff
@@ -117,17 +100,65 @@ def get_all_relevant_links(url):
             links = table.find_elements_by_tag_name("a")
             for link in links:
                 href = link.get_attribute("href")
-                if (
-                    "/tree/" in href or href.endswith(".py") or href.endswith(".ipynb")
-                ) and not "venv" in href:
+                if ("/tree/" in href) and not "venv" in href:
                     links_list.add(href)
                     queue.append(href)
+                elif (
+                    href.endswith(".py") or href.endswith(".ipynb")
+                ) and not "venv" in href:
+                    links_list.add(href)
+                    relevant_links_list.add(href)
         except:
             print("Error")
         bfs()
         print("Ended")
 
+    def process_files(link):
+        """
+        Processing each file using multiprocessing
+        :param link: link to the repository
+        :return: None
+        :rtype: None
+        """
+        options = Options()
+        options.headless = True
+        path_to_driver = glob.glob(
+            r"drivers\chromedriver\win32\92.0.4515.107\chromedriver.exe"
```

Comment:
drivers\chromedriver\win32\92.0.4515.107\chromedriver.exe the 92.0.45... is a version number it will change so make it dynamic else it will cause issues later.

Reply:
Thank you for bringing it to notice, drivers\chromedriver\win32\92.0.4515.107\chromedriver.exe has been changed to drivers\chromedriver\win32\*\chromedriver.exe so it is independent of the version